controllers/app.controllers.js

```diff
@@ -40,3 +40,15 @@ exports.getArticles = (req, res, next) => {
       next(err);
     });
 };
+
+exports.getCommentsByArticlesId = (req, res, next) => {
+  const id = req.params;
+  const articleId = id.article_id
```

Comment:
This is a little confusing. I would look into the notes on destructuring but I think the code you need is `const { articleId } = req.params` - this would give you the article_id variable to pass to selectArticlesByID